grpc.go

```diff
@@ -43,10 +45,13 @@ const (
 
 // CPU and Memory hotplug
 const (
+	cpuRegexpPattern = "cpu[0-9]*"
+	memRegexpPattern = "memory[0-9]*"
```

Comment:
Nit: shouldn't these be strictly?:

```
cpuRegexpPattern = "cpu[0-9][0-9]*"
memRegexpPattern = "memory[0-9][0-9]*"
```

Reply:
It's fine as long as files called `cpu` or `memory` are never created in that directory.